setup.py

```diff
@@ -6,8 +6,8 @@ def read(*parts):
     return codecs.open(path.join(path.dirname(__file__), *parts),
                        encoding="utf-8").read()
 
-setup(name='logstash_formatter',
-      version='0.5.15',
+setup(name='logstash-formatter',
+      version='1.5.15',
```

Comment:
Should be version 1.0.0. When you bump major everything else 0s out. When you bump MINOR revision 0s out. minor issue, but kinda nice to follow. You don't *have* to change it before merging but it's cool if you feel like it.